fluss-server/src/main/java/com/alibaba/fluss/server/kv/prewrite/KvPreWriteBuffer.java

```diff
@@ -164,11 +174,12 @@ private void update(Key key, Value value, long lsn) {
      * @param targetLogSequenceNumber the lower bound of the log sequence number truncated to.
      * @param truncateReason the reason to truncate
      */
-    public void truncateTo(long targetLogSequenceNumber, String truncateReason) {
-        LOG.info(
-                "Truncate the kv pre-write buffer to {} because we encounter error while building cdc logs: {}",
-                targetLogSequenceNumber,
-                truncateReason);
+    public void truncateTo(long targetLogSequenceNumber, TruncateReason truncateReason) {
+        if (truncateReason == TruncateReason.DUPLICATED) {
+            truncateAsDuplicatedCount.inc();
+        } else {
+            truncateAsErrorCount.inc();
```

Comment:
If the reason is not duplicated, which means is not unexpcted.. Can we print the error message? Otherwise, we have no way to know what exception happens in server.